cms_bootstrap/templatetags/bootstrap_tags.py

```diff
@@ -3,7 +3,11 @@
 
 from django.conf import settings
 from django import template
-from django.utils.text import mark_safe
+from django import VERSION as DJANGO_VERSION
+if DJANGO_VERSION < (2, 0):
```

Comment:
should be `< (1, 11)`  since `mark_safe` in Django-1.11 is already in `safestring`